tests/test_handlers.py

```diff
@@ -598,12 +598,36 @@ def test_am1_differences():
             assert 0
 
 
+def test_am1elf10_conformer_independence():
+    with resources.path("timemachine.testsystems.data", "ligands_40.sdf") as path_to_ligand:
+        suppl = Chem.SDMolSupplier(str(path_to_ligand), removeHs=False)
+    mols = [mol for mol in suppl][:2]  # truncate so that whole test is ~ 10 seconds
+
+    # need to assign so embedded molecules generated below
+    # have the correct stereochemistry
+    for mol in mols:
+        rdmolops.AssignStereochemistryFrom3D(mol, confId=0, replaceExistingTags=True)
```

Comment:
I changed this to use molecules at index 0, 2, 3 (each have chiral centers). This is what I used when developing the test, since the failure depends on EmbedMolecule randomly generating with the wrong chirality.